### DISCUSSION TITLE
How to merge large output to one result or doc?

### DISCUSSION THREAD

**Discussion by dotku** [FlowiseAI/Flowise#2863] [Q&A]
I have multiple output from ChatGPT, and I want to add then together to be a large output (could be a doc or chat result) how to make it?

I see there is sample from one output to 2nd output by using LLM chain, but didn't see how I can merge multiple output... the promptTemplate can take only one input :( 

![image](https://github.com/user-attachments/assets/50cf938f-3613-4f52-a964-14023e9be1a8)

**Comment by Artemnikov** [comment]
Sounds like a great idea!
For example if we could dump a workflow output into a document or chart that would be amazing

**Reply by dotku** [reply]
waiting for answer... otherwise, I just code it up without gui tool for now ^^!
